src/presenters/admin/indexing-notification-presenter.php

```diff
@@ -80,6 +80,12 @@ protected function get_message( $reason ) {
 			case Indexing_Notification_Integration::REASON_HOME_URL_OPTION:
 				$text = \esc_html__( 'Because of a change in your home URL setting, some of your SEO data needs to be reprocessed.', 'wordpress-seo' );
 				break;
+			case Indexing_Notification_Integration::REASON_CATEGORY_BASE_PREFIX:
+				$text = \esc_html__( 'Because of a change in your category base setting, some of your SEO data needs to be reprocessed.', 'wordpress-seo' );
+				break;
+			case Indexing_Notification_Integration::REASON_TAG_BASE_PREFIX:
+				$text = \esc_html__( 'Because of a change in your tag base setting, some of your SEO data needs to be reprocessed.', 'wordpress-seo' );
+				break;
```

Comment:
General note for future adjustments: I'm not sure this takes into consideration the case where users may update the category base and the tag base (and also the permalink structure) at the same time. I guess in that case the message will always be the one from the last case, referring to "tag base".

Reply:
Pinging @herregroen for some feedback 🙂 Should we create a new issue for this?

Reply:
Issue has been created: https://yoast.atlassian.net/browse/P3-240